### DISCUSSION TITLE
HelloGitHub Badge

### DISCUSSION THREAD

**Discussion by 521xueweihan** [chrxh/alien#104] [General]
Hi, We're thrilled to share that [your project](https://hellogithub.com/en/repository/d53e3c352f294f72a1bfd8f48ac0f866) has caught the attention of the HelloGitHub community and has been recognized for its merit. Your work is truly inspiring, and we'd like to invite you to participate in our [HelloGitHub Badge Program](https://hellogithub.com/en/repository/d53e3c352f294f72a1bfd8f48ac0f866/embed).

<img width="766" alt="image" src="https://github.com/user-attachments/assets/6648dffa-0da4-46b7-a9e6-dd73f5cbc53d">

By joining, you'll enjoy a few benefits that we believe will complement your project's journey:

1. **Community Acknowledgement**: The badge serves as a mark of distinction, indicating that your project has met the community's criteria for recommendation.
2. **Visibility Boost**: Wearing the badge can help increase the visibility of your project, potentially drawing the attention of new users and contributors.
3. **Easier Engagement**: The badge acts as a quick reference for users to understand your project, facilitating interaction through likes, comments, and bookmarks.
4. **Feedback Opportunities**: It opens the door for valuable feedback from a diverse user base, which can be instrumental in your project's growth and refinement.
5. **Notable Recognition**: Verified participants will receive a special identifier, highlighting their contributions within the community.

We invite you to consider joining the [HelloGitHub Badge Program](https://hellogithub.com/en/repository/d53e3c352f294f72a1bfd8f48ac0f866/embed) as a way to further engage with the community and enhance your project's presence. It's completely up to you, and we respect your decision either way.

Thank you for being part of this journey with us.

Warm regards,
The HelloGitHub Team

**Comment by chrxh** [comment]
Thanks, but I don't understand how putting an advertisement banner on the readme of my project should increase the project's visibility? (of course, it helps for your website)

**Reply by 521xueweihan** [reply]
1. The sorting algorithm of the HelloGitHub homepage will dynamically rank the project based on the likes, comments, ratings and other data. Wearing a badge on REAMDE can greatly improve the above data, so that the project can get a better ranking on the homepage and get more exposure
2. When using the search function on the website, certified projects will also be displayed first

At the same time, you can also see users from hellogithub in the traffic on github: https://github.com/chrxh/alien/graphs/traffic

The above is to hope that your open source project will get more attention, not forced promotion, the decision is yours. If my message has caused you trouble, then I apologize. Finally, thank you for your open source project, which made me discover the fun of open source and amazing programming skills.

**Reply by chrxh** [reply]
Thank you for the clarification and for featuring ALIEN in the latest volume!

**Reply by 521xueweihan** [reply]
Thank you for your support and claim. I have passed the claim review. If you encounter any problems, please contact me.

<img width="779" alt="image" src="https://github.com/user-attachments/assets/f6a20590-3d95-4d87-9f69-79802bef8fdb">
